scripts/globals/job_utils/dark_knight.lua

```diff
@@ -85,8 +85,10 @@ xi.job_utils.dark_knight.useConsumeMana = function(player, target, ability)
 end
 
 xi.job_utils.dark_knight.useDarkSeal = function(player, target, ability)
-    local merits = player:getMerit(xi.merit.DARK_SEAL)
-    player:addStatusEffect(xi.effect.DARK_SEAL, merits, 0, 60)
+    local power    = player:getMerit(xi.merit.DARK_SEAL) - 10
```

Comment:
Might want to leave a comment here about how the -10 is because the initial merit provides no cast time reduction and it's otherwise unusable without it, plus that each merit is 10% so 5/5 = -40%

Reply:
Comment added.